graph/multi/undirected.go

```diff
@@ -0,0 +1,250 @@
+// Copyright ©2014 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package multi
+
+import (
+	"fmt"
+
+	"gonum.org/v1/gonum/graph"
+	"gonum.org/v1/gonum/graph/internal/uid"
+)
+
+var (
+	ug *UndirectedGraph
```

Comment:
Same comment as above. Remove `ug` if not used.

Reply:
Ack